llama-index-integrations/llms/llama-index-llms-you/tests/test_llms_you.py

```diff
@@ -0,0 +1,7 @@
+from llama_index.core.base.llms.base import BaseLLM
+from llama_index.llms.you import YouLM
```

Comment:
I think this should be `You` instead of `YouLM`. Fixing this should then lead to all of our checks getting passed.